suite-native/feature-flags/src/featureFlagsSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { Platform } from 'react-native';
+
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { isDevelopOrDebugEnv } from '@suite-native/config';
+
+export interface FeatureFlagsState {
+    isDeviceConnectEnabled: boolean;
+}
+
+export type FeatureFlagsRootState = {
+    featureFlags: FeatureFlagsState;
+};
+
+export const featureFlagsInitialState: FeatureFlagsState = {
+    isDeviceConnectEnabled: Platform.OS === 'android' && isDevelopOrDebugEnv(),
+};
+
+export const featureFlagsPersistedKeys: Array<keyof FeatureFlagsState> = ['isDeviceConnectEnabled'];
+
+export const featureFlagsSlice = createSlice({
+    name: 'featureFlags',
+    initialState: featureFlagsInitialState,
+    reducers: {
+        setIsDeviceConnectEnabled: (state, { payload }: PayloadAction<boolean>) => {
```

Comment:
In my last `slice-related` PR you asked me to use something like `toggleIsDeviceConnectEnabled`, because then you do not have to use payload at all. Now I'm curios, is this case any different? Just trying to understand how you think about it, not requesting any change here.

Reply:
No, that's good point https://github.com/trezor/trezor-suite/pull/9840/commits/5f26d8a42f7657d579b87b15fda7576d67d39060